ci.tex

```diff
@@ -27,7 +27,7 @@ \subsubsection{Configuration \& Deployment}
 at the time DM was transitioning away from Buildbot.
 
 \href{https://puppet.com/}{Puppet} was selected as a configuration management (CM) tool as, at that time, it had the most sophisticated Jenkins
-management abilities among the CM tools surveyed via the \href{https://github.com/voxpupuli/puppet-jenkins}{\texttt{puppet-jenkins}} module.
+management abilities among the CM tools surveyed, via the \href{https://github.com/voxpupuli/puppet-jenkins}{\texttt{puppet-jenkins}} module.
```

Comment:
> Puppet was selected as a configuration management (CM) tool as, at that time, it had the most sophisticated Jenkins management abilities among the CM tools surveyed, via the puppet-jenkins module.

I think this is grammatically incorrect, and it's certainly confusing. Why not reorder to "Puppet, via the puppet-jenkins module, was selected..."?

Reply:
Puppet was selected as a configuration management (CM) tool as its puppet-jenkins module provided the most sophisticated Jenkins management abilities among the tools surveyed at that time.